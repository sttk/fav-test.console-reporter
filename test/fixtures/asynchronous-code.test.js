'use stirct';

describe('User', function() {
  describe('#save()', function() {
    it('should save without error', function(done) {
      var user = new User('Luna');
      user.save(function(err) {
        if (err) done(err);
        else done();
      });
    });
    it('should save without error (2)', function(done) {
      var user = new User('Luna');
      user.save(done);
    });
  });
});

function User(name) {
}
User.prototype.save = function(cb) {
  setTimeout(function() {
    cb();
  }, 1000);
};

