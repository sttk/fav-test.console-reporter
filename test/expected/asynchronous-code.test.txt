

  User
    #save()
      ✓ should save without error (1003ms)
      ✓ should save without error (2) (1005ms)


  2 passing (2s)

